### ISSUE TITLE
In mobile view sidebar remains in screen.

### ISSUE THREAD

**Issue by NitinChakrawarti** [ashutoshgithubs/iemhacks_DEV-G#79]
**Describe the bug**
In mobile devices sidebar remains in screen after clicking any of elements.

**To Reproduce**
open in mobile view.

**Expected behavior**
It must go off screen after clicking any elements of sidebar 

**Screenshots**

https://github.com/ashutoshgithubs/iemhacks_DEV-G/assets/149225785/1ae09fc2-b986-44ce-8834-b1de3434cd1a

**Additional context**

I can fix this using javascript and CSS 

@ashutoshgithubs please assign this issue to i want to work on this issue under GSSoC'24.

**Comment by ashutoshgithubs** [comment]
Hi @NitinChakrawarti , can you do it using JS framework - REACT & Tailwind CSS ? Let me know.

**Comment by NitinChakrawarti** [comment]
> Hi @NitinChakrawarti , can you do it using JS framework - REACT & Tailwind CSS ? Let me know.

Yes, for sure  @ashutoshgithubs  assign it to me, I will do that with mentioned tech stack

**Comment by VinayLodhi1712** [comment]
@ashutoshgithubs please assign this issue to me i also want to work on this issue under GSSoC'24.

**Comment by maha-sachin** [comment]
@NitinChakrawarti , 
Thank you for bringing this issue to our attention. I understand that the sidebar is not behaving as expected on mobile devices, as it remains on the screen after clicking any of its elements.

1. Ensure that the event handlers for clicking on sidebar elements are properly implemented and that they trigger the intended actions to hide the sidebar.
2. After making changes, thoroughly test the sidebar on mobile devices again to verify that the issue has been resolved.
3. Update relevant documentation or comments in the code to ensure that the behaviour of the sidebar is well-documented for future reference.


If you encounter any challenges during this process or need further assistance, feel free to reach out. We're here to help you resolve this issue and ensure a smooth user experience on mobile devices.
